### ISSUE TITLE
onImageForAnalysis for multi cam

### ISSUE THREAD

**Issue by lei-cao** [Apparence-io/CamerAwesome#333]
## Proposal
Thank you for providing the multi cam feature. Very important for my app. I am testing it, it's really cool.  But I realise that onImageForAnalysis is not support at this moment. 

*Describe your new feature*
Support onImageForAnalysis with multi cam.

**Comment by istornz** [comment]
I completely agree with @apalala-dev. It's important to acknowledge that while the multi-cam feature is still in beta, there are areas that need improvement.

I understand that image analysis is a great feature to implement but impact performance when multiple cameras are being displayed. Enabling it may not be feasible for now, especially on iOS where problems can arise when combining various use cases such as preview, video recording, and image analysis. It might be worth considering enabling it for the main camera initially.

**Comment by lei-cao** [comment]
enabling it for the main camera is enough for my case. Hope it could be implemented :) 👍
